### ISSUE TITLE
Length of input fields (simple form validation)

### ISSUE THREAD

**Issue by kugelbit** [shopware/platform#1926]
**Description**  
So i tried to register a shop-user with an vary long name (381 characters).  This was not successful. the front-end showed a side with an  "Something went wrong"-Error and the backend-log gave me an  Uncaught PHP Exception (1406 Data too long for column 'first_name' at row 1).. This is not vary nice for the Customer. Also our logistics partners only accepts names witch are shorter than 35 characters.  

So basically i need a simple form-validation where i can set maximum length of input fields. Also i want to inform the frontend user what went wrong (e.x. Name to long).

It would be beautiful if i can configure a form validation from the shopware-admin-backend.

Or maby there is a simple way when creating a shopware-theme ? Are there any tutorials about form-validation?

Thanks for your help

**Comment by MarcelSchmaeing** [comment]
Hello,

Within our company roadmap and work capacity, we try to address each bug or improvement request but admit that not each one will be resolved. To continue our culture of honesty and openness, we are closing this issue to focus on our roadmap on behalf of all Shopware users.

If you feel like this issue is still impacting you, please create a new issue and let us know.
Thank you again for your valuable feedback.
